Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -117,16 +119,26 @@ open class TopBannerViewController: UIViewController {
     
     private func showSecondaryChildren(completion: CompletionHandler? = nil) {
         statusView.isHidden = !statusView.isCurrentlyVisible
-        junctionView.isHidden = !junctionView.isCurrentlyVisible
-        lanesView.isHidden = !lanesView.isCurrentlyVisible
-        nextBannerView.isHidden = !nextBannerView.isCurrentlyVisible
         
-        UIView.animate(withDuration: 0.20, delay: 0.0, options: [.curveEaseOut], animations: { [weak self] in
-            guard let children = self?.informationChildren else {
-                return
+        if let tertiary = currentInstruction?.tertiaryInstruction {
+            lanesView.isHidden = !lanesView.isCurrentlyVisible
+            lanesView.update(for: currentInstruction)
```

Comment:
`LanesView`, `NextBannerView` and `JunctionView` already contain `update(for:)` method that checks for `tertiaryInstruction` or `quaternaryInstruction`. Can we use these methods directly here?